testnet/stacks-node/src/config.rs

```diff
@@ -833,6 +835,43 @@ impl Config {
             _ => false,
         }
     }
+
+    pub fn clean_working_dir(&self) {
+        let to_archive = fs::read_dir(&self.node.working_dir).unwrap()
+            .into_iter()
+            .filter(|r| r.is_ok())
+            .map(|r| r.unwrap().path().file_name().unwrap().to_str().unwrap().to_string())
+            .filter(|dir| !dir.ends_with("archive"))
+            .collect::<Vec<String>>();
+
+        let archive_name = format!("{}-archive", get_epoch_time_secs());
+        let archive_path = format!("{}/{}", &self.node.working_dir, archive_name);
+        fs::create_dir(&archive_path).unwrap();
+
+        for dir in to_archive.iter() {
+            let src = format!("{}/{}", &self.node.working_dir, dir);
+            let dst = format!("{}/{}", archive_path, dir);
+            fs::rename(src, dst).unwrap();
```

Comment:
Is there any sanitation/checking done to the configured `working_dir` before this? If I set `working_dir=/home` seems like this would move all my `/home` files?